projects/i18n/switch/options.ts

```diff
@@ -0,0 +1,19 @@
+import {inject, InjectionToken} from '@angular/core';
+import {LOCAL_STORAGE} from '@ng-web-apis/common';
+import {TuiLanguageName} from '@taiga-ui/i18n/interfaces';
+
+export type TuiLangChunkLoader = (lang: TuiLanguageName) => Promise<unknown>;
```

Comment:
Let's call it just `TuiLanguageLoader`. Nitpicking, but not a big fan of reducing `Language` to `Lang`. I believe somewhere in our internal codestyle there's an agreement not to do words reducing :)

Reply:
fixed